src/main/kotlin/com/example/iotmanagementdevice/mongock/changelogs/DeviceMigration.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.iotmanagementdevice.mongock.changelogs
+
+import com.example.iotmanagementdevice.model.MongoDevice
+import io.mongock.api.annotations.ChangeUnit
+import io.mongock.api.annotations.Execution
+import io.mongock.api.annotations.RollbackExecution
+import org.springframework.data.domain.Sort
+import org.springframework.data.mongodb.core.MongoTemplate
+import org.springframework.data.mongodb.core.index.Index
+import org.springframework.data.mongodb.core.index.IndexOperations
+
+@ChangeUnit(id = "deviceMigration", order = "001", author = "Taras Yurkovskyi")
+class DeviceMigration {
+
+    @Execution
+    fun applyDeviceMigration(mongoTemplate: MongoTemplate) {
+        if (!mongoTemplate.collectionExists(MongoDevice.COLLECTION_NAME)) {
+            mongoTemplate.createCollection(MongoDevice.COLLECTION_NAME)
+        }
+        createIndexes(mongoTemplate)
+    }
```

Comment:
I think you should drop collection you've created, if, for example, rollback is being performed on index creation, after collection was created